lib/integration/scenarios/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -4351,6 +4351,21 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
             , expectField #policyId (`shouldBe` (ApiT tokenPolicyId'))
             ]
 
+    it "TRANS_NEW_CREATE_12 - Cannot vote in Babbage" $ \ctx -> runResourceT $ do
+        noConway ctx "voting supported in Conway onwards and tested in API.Conway module"
```

Comment:
changed `noConway` using `_mainEra ctx >= ApiConway`

Reply:
>  noConway using _mainEra ctx >= ApiConway

Hm. Although if you change `noConway`, then this symmetry is odd:

<img width="674" alt="Skärmavbild 2024-02-22 kl  16 15 14" src="https://github.com/cardano-foundation/cardano-wallet/assets/304423/deacc552-68f8-41ab-a5d6-c537fd19ff4d">

Eh. It might be fine 🤷‍♂️ Or maybe it should be `preConway` or `noConwayOrLater` instead 🤔

Reply:
> This will break when a new era is added 

Although, this babbage only test is going to become untested very soon after the HF when we stop running integration tests in Babbage, so if it's around for the next era it may be deletable unless intended to be kept for theoretical manual runs 🤔